### PR TITLE
chore: spelling errors

### PR DESCRIPTION
**Pull request by burhan**
I noticed a few typos and spelling errors while reading through the code.

### CODE REVIEW

File: foundation/keystore/keystore.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ import (
 )
 
 // KeyStore represents an in memory store implementation of the
-// KeyStorer interface for use with the auth package.
+// KeyStore interface for use with the auth package.
```
Comment (ardan-bkennedy) [inline foundation/keystore/keystore.go:19]: This is wrong. It should stay KeyStorer. I am sure the interface is named that.

**Comment by ardan-bkennedy** [comment]
Fix the one comment and thanks for the PR. Is your name already in the CONTRIBUTORS file? If not, please add it in alphabetical order to the PR as well.

File: foundation/keystore/keystore.go (hunk 1 above)
Comment (bk-tfo) [reply foundation/keystore/keystore.go:19]: I based this on the module level comment which mentions `auth.KeyStore` interface. Honestly I was not able to find a reference for `Keystorer` interface.

Comment (ardan-bkennedy) [reply foundation/keystore/keystore.go:19]: The interface is called KeyStore. This comment should remain as is.

Comment (ardan-bkennedy) [reply foundation/keystore/keystore.go:19]: oh, i see you fixed it actually.

Comment (ardan-bkennedy) [reply foundation/keystore/keystore.go:19]: This should be called KeyStorer so I will fix that after this PR.

**Pull request merged**